### PR TITLE
Create section describing wallet passphrase & security in Lisk docs

### LINKED ISSUE #1847: Create a section describing wallet passphrases/security in Lisk docs
### Description
Create an overview section explaining the concepts of wallet passphrases and best practices for wallet security, etc.

### Location
`development`

#### Page location
`/beta/run-blockchain/account-management.adoc`

### Additional resources
https://www.doubloin.com/learn/12-vs-24-word-seed-phrase
https://support.metamask.io/hc/en-us/articles/360060826432-What-is-a-Secret-Recovery-Phrase-and-how-to-keep-your-crypto-wallet-secure

### PR DESCRIPTION
**Pull request by chris529**
Closes #1847

### CODE REVIEW

**Review by Ade-kola — APPROVED** [review]

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 1]
```diff
@@ -434,3 +438,52 @@ It is necessary to securely store or remember a secret recovery phrase and passw
...
+== Lisk wallet passphrase and security best practices
+
+Wallets play a crucial role, as they provide a secure means to store, send, and receive both LSK tokens and other tokens/coins.
+Wallets in general come in different forms, including a variety of software wallets, such as hot wallets, custodial/non-custodial and web wallets; to hardware wallets, and even paper wallets.
+They operate using a combination of public and private keys, ensuring the safety and authenticity of transactions on the Lisk blockchain.
+A more in-depth overview can be found here regarding the overall xref:{url_wallet_types}[types of wallets] available.
+
+=== Passphrases
+
+As previously discussed, Lisk accounts employ a 12-word xref:{url_mnemonic_passphrase}[Mnemonic Passphrase] to ensure robust security.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
As previously discussed, Lisk accounts employ a 12-word {url_mnemonic_passphrase}[Mnemonic Passphrase^] to ensure robust security.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 2]
```diff
@@ -434,3 +438,52 @@ It is necessary to securely store or remember a secret recovery phrase and passw
 |For users that solely require the key pair on a remote server, e.g., a validator node on a remote data center.
 Storing and backing up an encrypted file is necessary, but the user only needs to remember or securely store one password.
 |===
+
+== Lisk wallet passphrase and security best practices
+
+Wallets play a crucial role, as they provide a secure means to store, send, and receive both LSK tokens and other tokens/coins.
+Wallets in general come in different forms, including a variety of software wallets, such as hot wallets, custodial/non-custodial and web wallets; to hardware wallets, and even paper wallets.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
Wallets in general come in different forms, including a variety of software wallets, such as hot wallets, custodial/non-custodial, and web wallets; hardware wallets, and even paper wallets.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 3]
```diff
@@ -434,3 +438,52 @@ It is necessary to securely store or remember a secret recovery phrase and passw
...
+They operate using a combination of public and private keys, ensuring the safety and authenticity of transactions on the Lisk blockchain.
+A more in-depth overview can be found here regarding the overall xref:{url_wallet_types}[types of wallets] available.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
For a more in-depth overview regarding the wallets, see {url_wallet_types}[types of wallets^].
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 4]
```diff
@@ -434,3 +438,52 @@ It is necessary to securely store or remember a secret recovery phrase and passw
...
+Nevertheless, the BIP39 seed phrase relies on a finite pool of 2,048 words, which in turn restricts the potential permutations that can be produced.
+Conversely, a 24-word seed phrase comprises 256 bits of entropy, as each term encapsulates 11 bits, culminating in a total of 264 bits, including the checksum.
+
+The same rationale holds true for the 256 bits of entropy plus an extra 8 bits for the checksum.
+While a 12-word phrase offers greater memorability and ease of input, it concurrently becomes more susceptible to guessing or brute-force attacks by malicious actors.
+
+Given the mere 128 bits of entropy, deciphering a 12-word seed phrase would necessitate significantly less time and resources compared to a 24-word counterpart with 256 bits of entropy.
+Nevertheless, even in the face of heightened susceptibility to guessing, the task of brute-forcing remains nearly insurmountable.
+
+With 12 words offering a staggering 2^128 potential combinations, the probability of arriving at the correct combination is exceedingly remote.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
With 12 words offering a staggering 2^128^ potential combinations, the probability of arriving at the correct combination is exceedingly remote.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**